src/index.js

```diff
@@ -533,17 +533,45 @@ export default class RNPickerSelect extends PureComponent {
         );
     }
 
+    renderWeb() {
+        const { disabled, Icon, style, pickerProps } = this.props;
+        const { selectedItem } = this.state;
+
+        return (
+            <View style={[defaultStyles.viewContainer, style.viewContainer]}>
+                <Picker
+                    style={[
+                        Icon ? { backgroundColor: 'transparent' } : {}, // to hide native icon
```

Comment:
does this work on web?

Reply:
What in particular, the icon? The picker?

Everything I committed here leads to a working, style-able component for the web.

Reply:
The icon works, yes, though it should probably have some styling changes done to it, but Expo FontAwesome icons are loading.

Reply:
Getting rid of the native icon isn't necessary, removing for new commit